.github/workflows/checks.yml

```diff
@@ -304,4 +300,4 @@ jobs:
         pyomo build-extensions || python -c "from pyomo.contrib.pynumero.asl import AmplInterface; exit(0) if AmplInterface.available() else exit(1)"
     - name: Run pytest
       run: |
-        pytest --pyargs watertap -k 'not nf_dspmde.nf_ui'
+        pytest --pyargs watertap -k 'not (nf_dspmde.nf_ui or nf_dspmde.nf_with_bypass_ui)'
```

Comment:
I suppose the alternative GUI flowsheet code was also causing checks to fail, hence it's addition here. DOes that mean the UI code for these NF flowsheets still has issues to address?

Reply:
IIRC this is due to a limitation of the `watertap.ui.fsapi` layer for models that require the "good" (`ma27`-enabled) Ipopt to solve. If the model can be initialized without failures, specifying `requires_idaes_solver=True` kwarg is enough to skip the corresponding test without failures, but this doesn't work when the solve fails happens during initialization. Therefore, we have to exclude the entire test module using the `-k not ...` pytest flag.

Reply:
Note -- this only applies to the macOS tests